src/jdk.compiler/share/classes/com/sun/source/doctree/DocTreeVisitor.java

```diff
@@ -207,6 +207,16 @@ default R visitIndex(IndexTree node, P p) {
      */
     R visitLiteral(LiteralTree node, P p);
 
+    /**
+     * Visits a {@code MarkdownTree} node.
+     * @param node the node being visited
+     * @param p a parameter value
+     * @return a result value
+     *
+     * @since 21
+     */
+    R visitMarkdown(MarkdownTree node, P p);
```

Comment:
This is supposed to be a `default` method, right?

Reply:
Yes.